genmod/commands/analyze.py

```diff
@@ -11,226 +11,220 @@
     else:
-        variant_parser = VCFParser(infile = variant_file)
+        variant_parser = VCFParser(infile=variant_file)
     for variant in variant_parser:
-        genetic_models = variant['info_dict'].get('GeneticModels', None)
+        genetic_models = variant["info_dict"].get("GeneticModels", None)
         if genetic_models:
             for family_models in genetic_models:
-                family = family_models.split(':')[0]
+                family = family_models.split(":")[0]
                 families.add(family)
     return families
 
 
-def print_results(variant_dict, 
-                    outfile, 
-                    vcf_header,
-                    family_id, 
-                    score_key='CADD', 
-                    freq_key='1000G_freq', 
-                    mode = 'homozygote', 
-                    silent=False):
+def print_results(
+    variant_dict,
+    outfile,
+    vcf_header,
+    family_id,
+    score_key="CADD",
+    freq_key="1000G_freq",
+    mode="homozygote",
+    silent=False,
+):
     """Print the variants to a results file or stdout."""
-    
-    score_dict = {} # A dictionary with {variant_id: score}. Score is usually cadd score or rank score
+
+    score_dict = {}  # A dictionary with {variant_id: score}. Score is usually cadd score or rank score
     # for variant_id, variant in sorted(variant_dict.items(), key = lambda sort_key: float(sort_key[1]['info_dict'].get('CADD', '0')), reverse=True):
     column_width = 12
     length_of_output = 20
     for variant_id in variant_dict:
         # Get the score for each variant:
         max_score = max(
-                            [
-                                float(score) for score in 
-                                variant_dict[variant_id]['info_dict'].get(
-                                                                    score_key, 
-                                                                    '0')
-                            ]
-                        )
-        if mode == 'compound':
+            [float(score) for score in variant_dict[variant_id]["info_dict"].get(score_key, "0")]
+        )
+        if mode == "compound":
             # If we look at compounds we want to consider the combined score
-            family_compounds = compound_dict[variant_id]['info_dict'].get('Compounds', None)
-            if compounds:
+            family_compounds = variant_dict[variant_id]["info_dict"].get("Compounds", None)
+            if family_compounds:
                 for family in family_compounds:
-                    splitted_compounds = family.split(':')
-                    if splitted_compounds[0] == family_id:
-                        compounds = splitted_compounds[1].split('|')
-            
+                    split_compounds = family.split(":")
+                    if split_compounds[0] == family_id:
+                        compounds = split_compounds[1].split("|")
+
                 for variant_2_id in compounds:
```

Comment:
Shouldn't this be indented to be under the `if` statement above?

Reply:
That would be a style issue then? I don't mind terribly either way. For the sake of simplicity, I would just leave it as the author had it - the intention I believe being that once you have your "compounds" you go into the next loop, and there isn't really supposed to be a second family matching. I would perhaps have added a continue after the assignmnet. This function isn't really used I believe, so not worth a lot of effort. 😊